controllers/aws/s3-bucket.go

```diff
@@ -158,6 +165,23 @@ func (a *AwsClient) findIfDiffTags(tagsToUpdate map[string]string, tagsFromAws [
 	a.Log.V(1).Info("returend from find diff Tags", "isDiffTags", isDiffTags, "newTags", newTags)
 	return isDiffTags, newTags
 }
+func (a *AwsClient) isBucketManagedByOperator(bucketName string) bool {
+	tagsFromAws, err := a.s3Client.GetBucketTagging(&s3.GetBucketTaggingInput{Bucket: aws.String(bucketName)})
+	if err != nil {
+		a.Log.Error(err, "error from GetBucketTagging in checkIfOwnerBucketByTag")
+		return false
```

Comment:
this is not quite what i mean..
i think the function signature return value should be like before (bool, error).
but the error should return only in case of technical error.
so in case of aws timeout for example you should return:
```
false, error
```
in other cases you should return
```
true/false, nil
```